server/views/pages/vary-approve/approved.njk

```diff
@@ -11,20 +11,23 @@
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
             {{ govukPanel({
-                titleText: "The licence variation for " + licence.forename + " " + licence.surname + " has been approved."
+                titleText: "Licence variation for " + licence.forename + " " + licence.surname + " approved."
             }) }}
 
             <h2 class="govuk-heading-m">What happens next</h2>
             <p class="govuk-body" id="sent-to">
-                The licence variation for {{ licence.forename }} {{ licence.surname }} been approved.
+                The licence and post sentence supervision order variation for {{ licence.forename }} {{ licence.surname }} has been approved with your digital signature.
```

Comment:
Is there a check for the licenceType anywhere? Its possible a licence can can be AP only (which does not have a PSS (Post Sentence Supervision Order).

Usually a case is AP or AP_PSS

Reply:
I will check with Content Design and amend text accordingly.

Reply:
Content design has amended the sentence to not refer to the licence type

Reply:
Content changed to not refer to licence type